### DISCUSSION TITLE
Broken template and links

### DISCUSSION THREAD

**Discussion by Jvogie** [academicpages/academicpages.github.io#2183] [Q&A]
I forked the repo and have edited the navigation.yml and config.yml files but when I go to the deployed website none of my links work and the website looks like the picture below. Any solution for this?
![error screenshot](https://github.com/academicpages/academicpages.github.io/assets/115844460/de5e9a25-909f-42ea-bcde-a168f992388b)

**Comment by rjzupkoii** [comment]
This looks like a style sheet wasn't loaded by your web browser and can happen to an error in the configuration that prevents it from building correctly. Can you link to the repository that you are deploying from? Also, what web browser are you using to view the site?

**Comment by Jvogie** [comment]
I am using firefox. Here is the link to the repository https://github.com/Jvogie/website. Thanks for the help

**Comment by rjzupkoii** [comment]
It looks like there might be a misconfiguration in the [_config.yml](https://github.com/Jvogie/website/blob/master/_config.yml) - where are you deploying the website to?

**Comment by Jvogie** [comment]
I am just using github pages to deploy to https://jvogie.github.io/website/

**Comment by rjzupkoii** [comment]
That's likely the problem since the template is fairly strict about pathing due to how GitHub Pages works - if you update GitHub Pages to deploy to `jvogie.github.io` then it should work.
